src/rules/lua-truthiness.ts

```diff
@@ -1,4 +1,5 @@
-import { AST_NODE_TYPES, TSESTree } from "@typescript-eslint/experimental-utils";
+
```

Comment:
Empty line at the start of the file
(Does prettier not fix this? It looks like it is configured in .eslintrc, although I note that there's no CI flow for either of them in this repo)

Reply:
I'm unsure what happened here, my latest revision to these changes no longer has this issue.